### PR TITLE
OSDOCS-2363: Updating Kubernetes version to 1.22.1 for GA

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://issues.redhat.com/browse/OSDOCS-2363

Preview: https://deploy-preview-35958--osdocs.netlify.app/openshift-enterprise/latest/welcome/index.html (throughout doc set)

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@zhouying7780 Can you please review this update to bump from 1.22.0 -> 1.22.1 throughout the doc set? Thanks!

File: modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc [hunk 1]
```diff
@@ -87,9 +87,9 @@ Note the worker with the role `worker-rt` that contains the string `4.18.0-211.r
 NAME                               	STATUS   ROLES           	AGE 	VERSION                  	INTERNAL-IP
 EXTERNAL-IP   OS-IMAGE                                       	KERNEL-VERSION
 CONTAINER-RUNTIME
-cnf-worker-0.example.com	          Ready	 worker,worker-rt   5d17h   v1.22.0
+cnf-worker-0.example.com	          Ready	 worker,worker-rt   5d17h   v1.22.1
 128.66.135.107   <none>    	        Red Hat Enterprise Linux CoreOS 46.82.202008252340-0 (Ootpa)
-4.18.0-211.rt5.23.el8.x86_64   cri-o://1.22.0-90.rhaos4.8.git4a0ac05.el8-rc.1
+4.18.0-211.rt5.23.el8.x86_64   cri-o://1.22.1-90.rhaos4.8.git4a0ac05.el8-rc.1
```
Comment (mikemckiernan) [inline modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc:92]: Prolly doesn't matter, but my expectation is that the other values on this line change too.

File: modules/nodes-nodes-viewing-listing.adoc [hunk 2]
```diff
@@ -59,9 +59,9 @@ $ oc get nodes -o wide
...
-node1.example.com   Ready    worker   72m    v1.22.0   10.0.129.222   <none>        Red Hat Enterprise Linux CoreOS 48.83.202103210901-0 (Ootpa)   4.18.0-240.15.1.el8_3.x86_64   cri-o://1.22.0-30.rhaos4.8.gitf2f339d.el8-dev
-node2.example.com   Ready    worker   164m   v1.22.0   10.0.142.150   <none>        Red Hat Enterprise Linux CoreOS 48.83.202103210901-0 (Ootpa)   4.18.0-240.15.1.el8_3.x86_64   cri-o://1.22.0-30.rhaos4.8.gitf2f339d.el8-dev
+master.example.com  Ready    master   171m   v1.22.1   10.0.129.108   <none>        Red Hat Enterprise Linux CoreOS 48.83.202103210901-0 (Ootpa)   4.18.0-240.15.1.el8_3.x86_64   cri-o://1.22.1-30.rhaos4.8.gitf2f339d.el8-dev
+node1.example.com   Ready    worker   72m    v1.22.1   10.0.129.222   <none>        Red Hat Enterprise Linux CoreOS 48.83.202103210901-0 (Ootpa)   4.18.0-240.15.1.el8_3.x86_64   cri-o://1.22.1-30.rhaos4.8.gitf2f339d.el8-dev
+node2.example.com   Ready    worker   164m   v1.22.1   10.0.142.150   <none>        Red Hat Enterprise Linux CoreOS 48.83.202103210901-0 (Ootpa)   4.18.0-240.15.1.el8_3.x86_64   cri-o://1.22.1-30.rhaos4.8.gitf2f339d.el8-dev
```
Comment (mikemckiernan) [inline modules/nodes-nodes-viewing-listing.adoc:64]: My expectation is that other values change, but unlikely to be a customer concern.

File: modules/restore-determine-state-etcd-member.adoc [hunk 3]
```diff
@@ -70,7 +70,7 @@ $ oc get nodes -l node-role.kubernetes.io/master | grep "NotReady"
 .Example output
 [source,terminal]
 ----
-ip-10-0-131-183.ec2.internal   NotReady   master   122m   v1.22.0 <1>
+ip-10-0-131-183.ec2.internal   NotReady   master   122m   v1.22.1 <1>
 ----
 <1> If the node is listed as `NotReady`, then the *node is not ready*.
```
Comment (mikemckiernan) [inline modules/restore-determine-state-etcd-member.adoc:75]: The information payload on this callout seems low.

**Review by mikemckiernan — COMMENTED** [review]
/lgtm

File: modules/restore-determine-state-etcd-member.adoc (hunk 3 above)
Comment (bergerhoffer) [reply modules/restore-determine-state-etcd-member.adoc:75]: Possibly, but that's out of scope of this PR.

File: modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc (hunk 1 above)
Comment (bergerhoffer) [reply modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc:92]: Yeah possibly, but it's just example output, so we do the best we can. And without rerunning them all, we probably won't know what else to update.

**Pull request merged**